ducktape/cluster/json.py

```diff
@@ -23,11 +23,8 @@ class JsonCluster(Cluster):
     An implementation of Cluster that uses static settings specified in a cluster file.
     """
 
-    def __init__(self, cluster_json=None):
+    def __init__(self, cluster_json, *args, **kwargs):
         super(JsonCluster, self).__init__()
-        if cluster_json is None:
```

Comment:
@ewencp I removed it because I thought this branch is never used in the code. I just realized it is used, when the user specifies `--cluster=ducktape.cluster.json.JsonCluster`. 

I find that these two options, `--cluster=ducktape.cluster.json.JsonCluster` and `--use-cluster-info`, overlap with each other. Both provide the functionality to allow user to load the cluster info from a json file with hard-coded path on disk, which seems a little confusing. And this may also be useful to allow user to specify path to json file, instead of creating/editing the file with the hard-coded path.

Would it be better to provide a single option, say `--use-cluster-info=/path/to/jsonfile`, which will load the cluster info from that json file?